Makefile

```diff
@@ -20,6 +20,7 @@ tests:
 buildpacks.tests:
 invokers.tests:
 smoke-tests:
+template-tests:
```

Comment:
It's been a while since I've gone deep into `make`, so let me know if this change is unnecessary/wrong

Reply:
I think if anything, all the targets found here doesn't need to exists (at least not in this makefile). Though adding it here for consistency 👍